### DISCUSSION TITLE
Mask detection

### DISCUSSION THREAD

**Discussion by sudipg4112001** [CoinShift-Imaging/CoinShift-Imaging-Box#4] [Ideas]
Issue #2 
Hey, I am here to start the discussion.

**Comment by DARK-art108** [comment]
Yeah,Buddy are you using Faster-RCNN or YoloV3/V4 or SSD

**Reply by sudipg4112001** [reply]
I will try to do by using CNN or Yolo

**Reply by sudipg4112001** [reply]
But, you can suggest me something if you want

**Reply by DARK-art108** [reply]
Yeah the task you're talking about is can be done using YoloV5 for faster results,Try using that make sure you use colab notebooks or kaggle kernals for this Task,

Simple pipeline:
Train Yolo --> Custom mask images  ---> Inference/Test it. Make sure to train for high epochs for good bbox Prediction.

**Reply by sudipg4112001** [reply]
Okay, I will try using Yolo, but if I fail I would like to take other alternative to fulfill the requirements.

**Reply by DARK-art108** [reply]
https://youtu.be/10joRJt39Ns Refer to this vedio for refernce.

**Reply by DARK-art108** [reply]
Yeah buddy you can use any Model!! Always ping me up when you're facing some issues

**Reply by sudipg4112001** [reply]
Okay I will refer to the video. On the other hand, I would also prefer to keep other options open.

**Reply by sudipg4112001** [reply]
> Yeah buddy you can use any Model!! Always ping me up when you're facing some issues

Yeah sure, then assign me this one. So that I can start working

**Reply by DARK-art108** [reply]
Sorry, Buddy for not responding, I will begin assigning the issues from tomorrow onwards @sudipg4112001 from 8th March as per GSOC'21 rules
